src/content/contributions/add-ons/add-ons.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Add-ons
+---
+
+import { CheckmarkFilled16 } from '@carbon/icons-react';
+
+<AnchorLinks>
+
+- [Add-on repositories](#add-on-repositories)
```

Comment:
In the past we've referred to them as "Carbon Add-ons" versus including repos in the name, is referring to them as "add-on repos" intentional?

Reply:
I was under the impression add-ons were explicitly repositories. Will adjust.